ste/sender-blockBlobFromURL.go

```diff
@@ -134,6 +135,13 @@ func (c *urlToBlockBlobCopier) generatePutBlockFromURL(id common.ChunkID, blockI
 		// step 3: put block to remote
 		c.jptm.LogChunkStatus(id, common.EWaitReason.S2SCopyOnWire())
 
+		if (c.ChunkAlreadyUploaded(blockIndex)) {
+			c.jptm.LogAtLevelForCurrentTransfer(pipeline.LogDebug, fmt.Sprintf("Skipping chunk %d.", blockIndex))
```

Comment:
Do you think this should be more clear (i.e. skipping chunk since it was already uploaded)?